routers/web/repo/editor.go

```diff
@@ -80,6 +82,21 @@ func redirectForCommitChoice(ctx *context.Context, commitChoice, newBranchName,
 		}
 	}
 
+	prIndex := ctx.FormInt64("back_to_pr")
+	if prIndex > 0 {
+		// Redirect to the PR's files tab
+		link := ctx.Repo.RepoLink + fmt.Sprintf("/pulls/%d/files?refresh=%d", prIndex, time.Now().Unix())
+		diffHash := ctx.FormString("back_to_diff_hash")
+		r := regexp.MustCompile(`^[0-9a-f]{40}$`)
+		if len(diffHash) == 40 && r.MatchString(diffHash) {
+			link += "#diff-" + diffHash
+		}
+		// FIXME: Because PushToBaseRepo is async, we need to wait a bit before redirecting
+		time.Sleep(2 * time.Second)
+		ctx.Redirect(link, 302)
```

Comment:
`http.StatusXxx` but not a magic number. And 302 has been deprecated for long time.

Reply:
ec6ab8d2d5a8c2b01b7288fd1d7d58998e50de79